HLeithner/joomla-cms
# Prepared mod tags popular

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the module in all ways you can think of.

* create a frontend module
* create articles with tags
* check the tags in the module if its the same without the PR

### Expected result
Nothing changed.
